webui/react/src/components/ResourcePoolCard.stories.tsx

```diff
@@ -14,7 +15,7 @@ export default {
 export const Default = (): React.ReactNode => {
   return <ResourcePoolCard
     containerStates={[]}
-    resourcePool={resourcePools[Math.floor(Math.random()*resourcePools.length)]}
+    resourcePool={rps[Math.floor(Math.random()*rps.length)]}
```

Comment:
**suggestion:** [PR 1935](https://github.com/determined-ai/determined/pull/1935) will allow this line to be just:
```
    resourcePool={rps.random()}
```
If you merge this before that PR, lemme know and I can update the PR to update this portion during a rebase.